src/components/IRPeaksPanel.tsx

```diff
@@ -1,31 +1,117 @@
-import { ValueRenderers } from '..';
+import {
+  ColumnDef,
+  useReactTable,
+  getCoreRowModel,
+  flexRender,
+  SortingState,
+  getSortedRowModel,
+} from '@tanstack/react-table';
+import { useState } from 'react';
 
-import { Table } from './Table';
 import { IRPeak } from './context/data/DataState';
 
+type IRPeakPanelPreferences = Record<
```

Comment:
preferences could have more properties so better that is it an object and one of the property is 'columns'.

Now concerning the columns is it better to have an object or rather an array containing key, etc. After in the code you need to make some maps and it would probably be more intuitive on an array.

Reply:
So you perfer that we use an array containing key?

Reply:
i used the suggested type in the issue "array containing key"